hhnk_research_tools/logger.py

```diff
@@ -0,0 +1,163 @@
+# %%
+
+import inspect
+import logging
+import sys
+from logging import *  # noqa: F401,F403 # type: ignore
+from pathlib import Path
+
+LOGGING_CONFIG = {
+    "version": 1,
+    "disable_existing_loggers": True,
+    "loggers": {
+        "": {  # root logger
+            "level": "NOTSET",
+            "handlers": ["debug_console_handler"],  # , 'info_rotating_file_handler'],
+        },
+        "hhnk_research_tools": {
+            "level": "INFO",
+            "propagate": False,
+            "handlers": ["debug_console_handler"],
+        },
+        "fiona": {  # Quiet by default!
+            "handlers": ["null"],
+            "propagate": False,
+            "level": "ERROR",
+        },
+        "rasterio": {  # Quiet by default!
+            "handlers": ["null"],
+            "propagate": False,
+            "level": "ERROR",
+        },
+    },
+    "handlers": {
+        "null": {
+            "level": "INFO",
+            "class": "logging.NullHandler",
+        },
+        "debug_console_handler": {
+            "level": "DEBUG",
+            "formatter": "info",
+            "class": "logging.StreamHandler",
+            "stream": "ext://sys.stdout",
+        },
+        # 'info_rotating_file_handler': {
```

Comment:
waarom staat deze uit? Mogelijk is een `TimedRotatingFileHandler` ook wel handig, dan weet je waar je in het verleden moet zoeken.